structuredevent-model/src/test/java/com/sequenceiq/environment/parameter/dto/AwsParametersDtoTest.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.environment.parameter.dto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class AwsParametersDtoTest {
```

Comment:
This test is not testing what you meant. You should test that `AwsParameterDto` indeed contains an `AwsDiskEncryptionParametersDto` object, nothing more

Reply:
I have made the changes to the test case to check just for the object